cmd/talosctl/cmd/mgmt/cluster/create/bytesize/bytesize.go

```diff
@@ -0,0 +1,92 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package bytesize
+
+import (
+	"fmt"
+	"unicode"
+
+	"github.com/dustin/go-humanize"
+)
+
+// ByteSize implements pflag.Value interface and is meant to be used with flags specifying a size in bytes.
+// A value can be set before assigning to a flag to function as a default value.
+type ByteSize struct {
+	valBytes    uint64
+	val         string
+	defaultUnit string
+}
+
+// WithDefaultUnit creates a New ByteSize with a default unit.
+func WithDefaultUnit(unit string) *ByteSize {
+	return &ByteSize{defaultUnit: unit}
+}
+
+// New creates a New ByteSize without a default unit.
+func New() *ByteSize {
+	return &ByteSize{}
+}
+
+// Bytes returns the value in bytes.
+func (bs *ByteSize) Bytes() uint64 {
+	return bs.valBytes
+}
+
+// Megabytes returns the value in megabytes.
+func (bs *ByteSize) Megabytes() uint64 {
+	return bs.valBytes / (1000 * 1000)
+}
+
+// Gigabytes returns the value in gigabytes.
+func (bs *ByteSize) Gigabytes() uint64 {
+	return bs.valBytes / (1000 * 1000 * 1000)
+}
+
+// IMegabytes returns the value in binary megabytes.
```

Comment:
Maybe worth naming it Mebibyte? Also same for other `I`-prefixed methods.

Reply:
Ah didn't know they were called that. Thanks!